README.MD

```diff
@@ -121,3 +124,49 @@ Swagger Documentation is provided in the controllers of the routes and lives at
 -   [Local API Specs](https://localhost:3000/v1/docs/)
 -   [Staging API Specs](https://dev.api.thx.network/v1/docs/)
 -   [Production API Specs](https://api.thx.network/v1/docs/)
+
+## 8. Database Schema Migration
+
+### 8.1 To add a new migration script, use the following CLI command
+
+```
+migrate-mongo create name-of-my-script
```

Comment:
@JustinaMary Please also add `migrate-mongo` as a dev dependency to the package.json.

Reply:
Sure